source/release-notes/2.5.0.rst

```diff
@@ -15,7 +15,7 @@ performance.
 New Features
 ================================================================================
 
-* :jirabug:`PMM-5042` and :jirabug:`PMM-5272`: PMM Server can now connect to MySQL instances by specifying a UNIX socket. This can be done with a new ``--socket`` option of the ``pmm-admin add mysql`` command.
+* :jirabug:`PMM-5042` and :jirabug:`PMM-5272`: PMM Server can now connect to MySQL instances by specifying a UNIX socket. This can be done with a new ``--socket`` option of the ``pmm-admin add mysql`` command. (Note: Updates to both PMM Client and PMM Server were done to allow UNIX socket connections.)
```

Comment:
```suggestion
* :jirabug:`PMM-5042` and :jirabug:`PMM-5272`: PMM can now connect to MySQL instances by specifying a UNIX socket. This can be done with a new ``--socket`` option of the ``pmm-admin add mysql`` command. (Note: Updates to both PMM Client and PMM Server were done to allow UNIX socket connections.)
```